vArmour_AppController_Connector.md

```diff
@@ -0,0 +1,22 @@
+# Connect your vArmour Application Controller to Azure Sentinel
```

Comment:
Move this file to solutions, suggested paths "Azure-Sentinel/Solutions/vArmour" or "Azure-Sentinel/Solutions/vArmour/Data Connectors"

Reply:
Relocated to "Azure-Sentinel/Solutions/vArmour/Data Connectors"